example/rater/generated_answer.ipynb

```diff
@@ -0,0 +1,355 @@
+      "                        'reject']}],\n",
+      "  'root': <uniflow.node.Node object at 0x7fc7ec328640>},\n",
+      " {'output': [{'average_score': 0.0,\n",
+      "              'error': 'No errors.',\n",
+      "              'majority_vote': 'equivalent',\n",
+      "              'response': ['explanation: Both answers are equally good as they '\n",
+      "                           'both correctly state that Vitamin C is '\n",
+      "                           'water-soluble.\\n'\n",
+      "                           'label: Equivalent',\n",
+      "                           'explanation: The grounding answer is more specific '\n",
+      "                           'in stating that Vitamin C is \"very\" water-soluble, '\n",
+      "                           'while the generated answer simply states that it '\n",
+      "                           'can be dissolved in water well. However, both '\n",
+      "                           'answers convey the same idea and are equally '\n",
+      "                           'correct.\\n'\n",
+      "                           'label: Equivalent',\n",
+      "                           'explanation: Both answers are equivalent as they '\n",
+      "                           'correctly state that Vitamin C is water-soluble.\\n'\n",
+      "                           'label: Equivalent',\n",
+      "                           'explanation: Both answers are equally good as they '\n",
+      "                           'both correctly identify Vitamin C as a '\n",
+      "                           'water-soluble vitamin.\\n'\n",
+      "                           'label: Equivalent',\n",
+      "                           'explanation: Both answers are equally good as they '\n",
+      "                           'both correctly identify Vitamin C as a '\n",
+      "                           'water-soluble vitamin.\\n'\n",
+      "                           'label: Equivalent'],\n",
+      "              'scores': [0.0, 0.0, 0.0, 0.0, 0.0],\n",
+      "              'votes': ['equivalent',\n",
+      "                        'equivalent',\n",
+      "                        'equivalent',\n",
+      "                        'equivalent',\n",
+      "                        'equivalent']}],\n",
+      "  'root': <uniflow.node.Node object at 0x7fc7d59fe740>}]\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "output = client.run(data)\n",
+    "pprint.pprint(output)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "data 0 has majority vote accept and average score 1.0\n",
+      "data 1 has majority vote reject and average score -1.2\n",
+      "data 2 has majority vote equivalent and average score 0.0\n"
```

Comment:
Is there a way to highlight `accept`, `reject` and `equivalent` and their scores? e.g. 

"data 1 has majority vote `REJECT` and average score `-1.2` over five-time LLMs' simulations.